docs/fr/developers/01_First_steps.md

```diff
@@ -1,20 +1,101 @@
-# Configurer son environnement
+# Configurer son environnement (Docker)
 
-**TODO**
+FreshRSS est construit en PHP et utilise le framework Minz. Les dépendances
+sont directement inclus dans le code source, donc vous n'avez pas besoin de
+Composer.
 
-## Docker
+Il existe plusieurs façons de configurer votre environnement de
+développement. La méthode la plus simple et la plus supportée est basée sur
+Docker, lequel est la solution documentée ci-dessous. Si vous avez déjà un
+environnement PHP fonctionnel, vous n'en avez probablement pas besoin.
 
-Le développement et le deploiement de FreshRSS peuvent se faire [via Docker](https://github.com/FreshRSS/FreshRSS/tree/dev/Docker).
+Nous supposons ici que vous utilisez une distribution GNU/Linux, capable
+d'exécuter Docker. Sinon, vous devrez adapter les commandes en conséquence.
+
+Les commandes qui suivent doivent être exécutées dans une console. Ils
+commencent par `$` quand les commandes doivent être exécutées en tant
+qu'utilisateur normal, et par `#` quand elles doivent être exécutées en tant
+qu'utilisateur root. Vous n'avez pas besoin de taper ces caractères. Un
+chemin d'accès peut être indiqué devant ces caractères pour vous aider à
+identifier où ils doivent être exécutés. Par exemple, `app$ echo 'Hello
+World'` indique que vous devez exécuter la commande `echo` dans le
+répertoire `app/`.
+
+Tout d'abord, vous devez installer
+[Docker](https://docs.docker.com/install/linux/docker-ce/ubuntu/).
+
+Lorsque vous avez terminé, clonez le référentiel avec :
+
+```console
+$ git clone https://github.com/FreshRSS/FreshRSS.git
+$ cd FreshRSS
+```
+
+Notez que, si vous voulez contribuer, vous devez d'abord forker le
+référentiel et cloner votre fork au lieu de la « root ». Adaptez les
+commandes en conséquence.
+
+Ensuite, la seule commande que vous devez savoir est la suivante :
+
+```console
+$ make start
+```
+
+Cela peut prendre un certain temps pendant que Docker télécharge l'image. Si
+votre utilisateur n'est pas dans le groupe `docker', vous devrez ajouter
+`sudo` à la commande.
+
+**Vous pouvez maintenant accéder à FreshRSS à [http://localhost:8080](http://localhost:8080).** Suivez simplement le processus d'installation et sélectionnez la base de données SQLite.
+
+Vous pouvez arrêter les conteneurs en tapant <kbd>Control</kbd> + <kbd>c</kbd> ou avec la commande suivante, dans un autre terminal:
+
+```console
+$ make stop
+```
+
+Si la configuration vous intéresse, les commandes `make' sont définies dans
+le fichier [`Makefile`](/Makefile).
+
+Si vous avez besoin d'utiliser une autre image de tag (par défaut
+`dev-alpine'), vous pouvez définir la variable d'environnement `TAG` :
+
+```console
+$ TAG=dev-arm make start
+```
+
+Vous pouvez trouver la liste complète des tags disponibles [sur le hub
+Docker](https://hub.docker.com/r/freshrss/freshrss/tags).
+
+Vous voudrez peut-être reconstruire l'image du Docker localement. Tu peux le
+faire avec :
```

Comment:
Si vous voulez construire l'image Docker, vous pouvez lancer la commande suivante :

Reply:
I'll also change the English.

> You might want to rebuild the Docker image locally. You can do it with:

* If you want to build the Docker image yourself, you can use the following command: